Assets/Mirror/Editor/Weaver/Readers.cs

```diff
@@ -0,0 +1,257 @@
+using System;
+using Mono.Cecil;
+using Mono.Cecil.Cil;
+
+
+namespace Mirror.Weaver
+{
+    public static class Readers
+    {
+        const int MaxRecursionCount = 128;
+
+        public static MethodReference GetReadFunc(TypeReference variable, int recursionCount = 0)
+        {
+            if (recursionCount++ > MaxRecursionCount)
```

Comment:
you increase recursioncount by 1 in line 48 already. ++ probably not needed here?

Reply:
good catch @vis2k